src/events/OmniboxEvents.ts

```diff
@@ -80,4 +84,8 @@ export class OmniboxEvents {
    * This event is only triggered by standard ML-powered query suggestions, and not {@link AnalyticsSuggestions} or {@link FieldSuggestions}.
    */
   public static querySuggestSuccess = 'querySuggestSuccess';
+  /**
+   * Triggered by the [`Omnibox`]{@link Omnibox} component when a query suggestion is hovered by the mouse or selected with the keybord.
```

Comment:
`keyboard`

Also, this is pretty confusing, as the name is about "selection", but then it says it's triggered when a selection is hovered ?

There should probably be two different event: 

`querySuggestionHover`, when a suggestion is hovered through mouse or keyboard.
`querySuggestionSelection` when a suggestion is selected through mouse or keyboard.